x/campaign/types/message_burn_vouchers_test.go

```diff
@@ -0,0 +1,65 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/spn/testutil/sample"
+	"github.com/tendermint/spn/x/campaign/types"
+)
+
+func TestMsgBurnVouchers_ValidateBasic(t *testing.T) {
+	tests := []struct {
+		name string
+		msg  types.MsgBurnVouchers
+		err  error
+	}{
+		{
+			name: "invalid address",
+			msg: types.MsgBurnVouchers{
+				Creator:    "invalid_address",
+				CampaignID: 0,
+				Vouchers:   sample.Coins(),
+			},
+			err: sdkerrors.ErrInvalidAddress,
+		},
+		{
+			name: "valid address",
```

Comment:
```suggestion
			name: "valid message",
```
We can also move this one first